app/javascript/components/Orientation.vue

```diff
@@ -22,7 +22,10 @@
       </div>
       <div class='column'>
         <div>
-          <h2 class='subtitle is-4'><a href='/external_resources/new'>Is there an outside resource we should know about?</a></h2>
```

Comment:
Changed this from one sentence to match the format of the others and go directly to the `new_external_resource` endpoint.